test/workers/zync_worker_test.rb

```diff
@@ -50,6 +50,34 @@ class ZyncWorkerTest < ActiveSupport::TestCase
     end
   end
 
+  test 'deleted application recreates dependencies' do
```

Comment:
Just a small technicality but I think it's not actually "_recreates_ dependencies", but rather "_rebuilds_ dependencies", no?

Reply:
Right, thanks! 👍